src/utils/constants.ts

```diff
@@ -1,14 +1,37 @@
+import { ETHEREUM_NETWORK } from 'src/config/networks/network.d'
+
 export const APP_ENV = process.env.REACT_APP_ENV
 export const NODE_ENV = process.env.NODE_ENV
 export const NETWORK = process.env.REACT_APP_NETWORK?.toUpperCase() || 'RINKEBY'
+export const INTERCOM_ID = APP_ENV === 'production' ? process.env.REACT_APP_INTERCOM_ID : 'plssl1fl'
 export const GOOGLE_ANALYTICS_ID = {
```

Comment:
I think we already discussed this, but just to be sure, I think would be a good idea to create a ticket to discuss a better way to make this config scalable without us to edit this file every time we need to support another network on analytics

Reply:
Actually, yesterday I added a description in the setup/deployment/CI task on that regard: https://github.com/gnosis/safe-react/issues/1421#issue-712247792